client/src/components/Banner/Banner.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Banner.css";
+
+const Banner = props => {
+  let classlist = "Banner";
+  return <div className={classlist} style={{
+    backgroundImage: `url(${props.background})`,
+    height: `${props.Height}`, fontSize: `${props.FontSize}`, lineHeight: `${props.LineHeight}`,
+    textAlign: `${props.TextAlign}`, display: `${props.display}`, flexDirection: `${props.flexdirection}`,
+    justifyContent: `${props.justifycontent}`, alignItems: `${props.alignitems}`
+  }}>
```

Comment:
Do you need to pass all these props down? This seems excessive. Remember that you can apply styling to the children directly!

Reply:
We definitely want to apply the background image this way